### ISSUE TITLE
Review truncating long translation strings

### ISSUE THREAD

**Issue by MrBurrBurr** [JustArchiNET/ASF-ui#304]
![jjgvi](https://user-images.githubusercontent.com/31552675/51794293-ce9f3d00-21cf-11e9-978a-2e61dd8b7a03.png)

In some places we should think about truncating strings if they get to long.

**Comment by JustArchi** [comment]
Only if you add proper tooltips on them, since otherwise newcomers will have no clue what those are.

**Comment by MrBurrBurr** [comment]
Yes, thats what I had in mind. Every string that gets truncated will also get a `title` containing the full string.
